src/main/io/dataEdition.h

```diff
@@ -0,0 +1,220 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Authors:
+ * Dominic Clifton/Hydra
+ * Carsten Giesen
+ * Adam Majerczyk (majerczyk.adam@gmail.com)
+ * Texmode add-on by Michi (mamaretti32@gmail.com)
+ *
+ *  Created on: december 2015
+ *  Author: Pierre Archambault
+ */
+
```

Comment:
I find it useful to separate public interface (anyone can include) and mostly internal interface (implementation details that must be shared between closely related .c files. I use "<x>_impl.h" for internal interface.
Don't pollute global namespace unless necessary and try to prefix macros/functions so that it is obvious who declared them